SoObjects/Appointments/SOGoAppointmentObject.m

```diff
@@ -1670,10 +1670,13 @@ - (NSException *) changeParticipationStatus: (NSString *) _status
               if ([event isAttendee: [[delegate email] rfc822Email]])
                 ex = [NSException exceptionWithHTTPStatus: 409
                                                    reason: @"delegate is a participant"];
-              else if ([SOGoGroup groupWithEmail: [[delegate email] rfc822Email]
-                                        inDomain: [ownerUser domain]])
-                ex = [NSException exceptionWithHTTPStatus: 409
-                                                   reason: @"delegate is a group"];
+              else {
+                NSDictionary *dict;
+                dict = [[SOGoUserManager sharedUserManager] contactInfosForUserWithUIDorEmail: [[delegate email] rfc822Email]];
```

Comment:
Respect the domain by using `[SOGoUserManager contactInfosForUserWithUIDorEmail:inDomain:]`.

Reply:
Resolved in commit 948c89f